src/app/main.cpp

```diff
@@ -376,6 +380,14 @@ void sigAbnormalHandler(int signum)
 #endif // defined(Q_OS_UNIX) || defined(STACKTRACE_WIN)
 
 #ifndef DISABLE_GUI
+void setupDpi()
+{
+#if defined(Q_OS_UNIX)
```

Comment:
now we support portable mode, might be possible to control setting `WindowsArguments = dpiawareness=0 or 1` in `qt.conf` here.
so maybe it's a good idea to leave it as-is.